lightning/src/ln/peer_handler.rs

```diff
@@ -757,7 +758,19 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	fn enqueue_message<M: wire::Type>(&self, peer: &mut Peer, message: &M) {
 		let mut buffer = VecWriter(Vec::with_capacity(2048));
 		wire::write(message, &mut buffer).unwrap(); // crash if the write failed
-		log_trace!(self.logger, "Enqueueing message {:?} to {}", message, log_pubkey!(peer.their_node_id.unwrap()));
+
+		match message.type_id() {
+			msgs::Ping::TYPE |
+			msgs::Pong::TYPE |
```

Comment:
Should these be included given they are not gossip?

Reply:
Yep, removed `Ping`/`Pong` and added `ChannelUpdate` with https://github.com/rust-bitcoin/rust-lightning/pull/1145/commits/753889b036d52eba9a95a57a610771d602cf8f74.